core/server/worker/src/main/java/alluxio/worker/netty/BlockWriteHandler.java

```diff
@@ -103,9 +120,6 @@ public BlockPacketWriter(
     @Override
     protected void completeRequest(BlockWriteRequestContext context, Channel channel)
         throws Exception {
-      if (context == null) {
```

Comment:
Is it guaranteed that context is not null?

Reply:
yes. talked offline